### ISSUE TITLE
CPX_PARAM_SCRIND works, CPXPARAM_ScreenOutput doesn't

### ISSUE THREAD

**Issue by dourouc05** [JuliaOpt/CPLEX.jl#229]
I just tried to make CPLEX shut up, so I went to the documentation to find the corresponding parameter (https://www.ibm.com/support/knowledgecenter/SSSA5P_12.6.3/ilog.odms.cplex.help/CPLEX/Parameters/topics/SCRIND.html). The old parameter's name is CPX_PARAM_SCRIND, and the new one is CPXPARAM_ScreenOutput. Both are defined in CPLEX.jl and mapped to 1035 (https://github.com/JuliaOpt/CPLEX.jl/blob/master/src/cpx_params_1280.jl). However, CPX_PARAM_SCRIND actually removes all output from CPLEX, whereas CPXPARAM_ScreenOutput does nothing. 

Here is my code to instantiate CPLEX within JuMP: 

```Model(with_optimizer(CPLEX.Optimizer, CPX_PARAM_SCRIND=0))```

**Comment by odow** [comment]
Is it because we turn `CPX_PARAM_SCRIND` on by default?
https://github.com/JuliaOpt/CPLEX.jl/blob/bb3f45c236fb0afb2fa2ed75c448e17b9a64a61e/src/MOI_wrapper.jl#L49

**Comment by dourouc05** [comment]
I'd say very likely!

**Comment by leejm516** [comment]
I found that changing solver parameters is very confusing with `MathOptInterface`.



```
model = CPLEX.Optimizer()
# Solver parameters
cpxParams = Dict("CPX_PARAM_SCRIND" => 0,
                "CPXPARAM_Simplex_Tolerances_Feasibility" => 1e-9,
                "CPXPARAM_Simplex_Tolerances_Optimality" => 1e-9,
                "CPXPARAM_Network_Tolerances_Optimality" => 1e-9,
                "CPXPARAM_Network_Tolerances_Feasibility" => 1e-9,
                "CPXPARAM_Barrier_ConvergeTol" => 1e-9)
model.params = cpxParams # This is not enough
MOI.empty!(model) # This must be executed
```
Both of the source code and docs are still immature for the present, which will be solved in the future.

**Comment by odow** [comment]
Closed by #237
